### ISSUE TITLE
Where does Helm store registry credentials ?

### ISSUE THREAD

**Issue by Ji993** [helm/helm#13571]
I can't find the filesystem location where Helm stores registry credentials.
`helm registry login registry-1.docker.io`

Output of `helm version`:

> version.BuildInfo{Version:"v3.16.3", GitCommit:"cfd07493f46efc9debd9cc1b02a0961186df7fdf", GitTreeState:"clean", GoVersion:"go1.22.7"}

Output of `kubectl version`:

> Client Version: v1.29.11
> Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
> Server Version: v1.29.11

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
N/A

**OS:** Ubuntu 22
**Duplicate of #11526 because no one answered back in 2022.**

**Comment by tpresa** [comment]
On my Mac I can see it under `~/Library/Preferences/helm/registry/config.json`. I imagine you could find yours in `~/.config/helm/registry/config.json`

**Comment by Ji993** [comment]
You're right. Credentials are stored in `~/.config/helm/registry/config.json` by default.

I see why I wasn't able to find them. I set the `XDG_CONFIG_HOME` to a specific directory so it put the credentials there.
